arbor/util/rangeutil.hpp

```diff
@@ -261,12 +263,15 @@ template <
     typename Compare = std::less<Value>
 >
 std::pair<Value, Value> minmax_value(const Seq& seq, Compare cmp = Compare{}) {
+    using std::begin;
+    using std::end;
+
     if (util::empty(seq)) {
         return {Value{}, Value{}};
     }
 
-    auto i = std::begin(seq);
-    auto e = std::end(seq);
+    auto i = begin(seq);
+    auto e = end(seq);
```

Comment:
It is overkill to have `using std::begin` when it is used once (ditto `std::end`).